src/Stryker.Core/Stryker.Core/Mutants/MutantOrchestrator.cs

```diff
@@ -60,21 +60,29 @@ public IEnumerable<Mutant> GetLatestMutantBatch()
         /// <returns>Mutated node</returns>
         public SyntaxNode Mutate(SyntaxNode currentNode)
         {
-            if (currentNode is StatementSyntax && currentNode.Kind() != SyntaxKind.Block)
+            if (GetExpressionSyntax(currentNode) is var expressionSyntax && expressionSyntax != null) {
+                // The mutations should be placed using a ConditionalExpression
+                var expression = GetExpressionSyntax(currentNode);
+                SyntaxNode ast = currentNode;
+                foreach (var mutant in FindMutants(expression))
+                {
+                    _mutants.Add(mutant);
+                    var mutatedNode = ApplyMutant(expression, mutant);
+                    ast = ast.ReplaceNode(expression, MutantPlacer.PlaceWithConditionalExpression(expression, mutatedNode, mutant.Id));
+                }
+                return ast;
+            }
+            else if (currentNode is StatementSyntax && currentNode.Kind() != SyntaxKind.Block)
```

Comment:
(not caused by this PR)

Won't the BlockSyntax check prevent mutators like the [Block mutator](https://github.com/stryker-mutator/stryker-handbook/blob/master/mutator-types.md#block-statement) from working correctly?

Reply:
For now skipping block statements makes this work, else the mutation process crashes. 

Also for the block mutator: I guess the Block Mutator will be a special kind. It will produce a compile error outside of the mutationIf when the block removes a return statement. The framework will not be able to rollback such an error. This is why I chose to not create an issue for that mutator yet. I'll create an issue for it now. Maybe someone knows how to fix that problem!

Reply:
Ah, I hadn't considered that.  An issue makes sense.